app/src/main/java/com/lightricks/feedexercise/network/DataTransferObjects.kt

```diff
@@ -3,3 +3,20 @@ package com.lightricks.feedexercise.network
 /**
  * todo: add Data Transfer Object data class(es) here
  */
+
+data class TemplatesMetadataItem(
+        val configuration: String,
+        val id: String,
+        val isNew: Boolean,
+        val isPremium: Boolean,
+        val templateCategories: List<String>,
+        val templateName: String,
+        val templateThumbnailURI: String
+)
+
+data class MetadataItemsList(
+        val templatesMetadata: List<TemplatesMetadataItem>
+)
```

Comment:
move it to be an internal class of the `TemplatesMetadataItem` class since they are related.